photo/tests/test_mutations/test_collection.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+from django.test import TestCase
+
+from photo.schema import schema
+from photo.tests.factories import CollectionFactory, PictureFactory, UserFactory
+from .mutation_file import (
+    collection_add_picture_mutation,
+    collection_creation_mutation,
+    collection_update_mutation,
+)
+
+
+class CollectionTest(TestCase):
+    def setUp(self):
+        self.newUser = UserFactory(user_profile_picture=True)
+        self.newPictures = PictureFactory.create_batch(10, user=self.newUser)
+
+    @pytest.mark.asyncio
+    async def test_create_one(self):
+        mutation = collection_creation_mutation
```

Comment:
this is not necessary, there's no clear advantage on assigning it to a new variable